pkg/controller/apicurioregistry/svc/patcher/patcher_kube.go

```diff
@@ -53,24 +60,44 @@ func (this *KubePatcher) patchApicurioRegistry() { // TODO move to separate file
 			panic("Unsupported operation.")
 		},
 		func(namespace common.Namespace, name common.Name, data []byte) (interface{}, error) {
-			return this.ctx.RequireService(svc.SVC_CLIENTS).(*client.Clients).CRD().PatchApicurioRegistry(namespace, name, data)
+			return this.clients.CRD().PatchApicurioRegistry(namespace, name, data)
 		},
 		func(value interface{}) common.Name {
 			return common.Name(value.(*ar.ApicurioRegistry).GetName())
 		},
 	)
 }
 
+func (this *KubePatcher) patchApicurioRegistryStatus() {
```

Comment:
Why patch status explicitly instead of using the status CF?

Reply:
still needs review